source-hive/src/main/java/org/apache/kylin/source/hive/BeelineHiveClient.java

```diff
@@ -27,6 +27,7 @@
 import java.sql.Statement;
 import java.util.List;
 import java.util.Properties;
+import java.io.*;
```

Comment:
I think "import *" is not allowed in Kylin's static code analysis. Could you please change to use the detailed class import?

Reply:
You can set up your kylin development environment by following
http://kylin.apache.org/development/dev_env.html